vendor/k8s.io/kubernetes/pkg/util/mount/mount.go

```diff
@@ -37,6 +41,9 @@ type Interface interface {
 	// IsLikelyNotMountPoint determines if a directory is a mountpoint.
 	// It should return ErrNotExist when the directory does not exist.
 	IsLikelyNotMountPoint(file string) (bool, error)
+	// GetDeviceNameFromMount finds the device name by checking the mount path
+	// to get the global mount path which matches its plugin directory
+	GetDeviceNameFromMount(mountPath, pluginDir string) (string, error)
```

Comment:
Origin is required to compile on all Linux platforms that have a build target (32, 64, arm64, and s390 eventually).